fortigate-autoscale/package.json

```diff
@@ -10,7 +10,7 @@
     ],
     "scripts": {
         "test": "echo \"Error: no test specified\" && exit 1",
-        "build": "../node_modules/.bin/tsc -b"
+        "build": "npx tsc -b"
```

Comment:
https://docs.npmjs.com/cli/v7/using-npm/scripts#path

tsc should already be added to your path, why do you need npx?

Reply:
done

Reply:
is npx really needed?